.github/workflows/node.js.yml

```diff
@@ -34,3 +34,7 @@ jobs:
       uses: coverallsapp/github-action@master
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
+
+    - run: npm run docker:test
+      env:
+        SOURCE_URL: ${{ secrets.SOURCE_URL }}
```

Comment:
might not be able to use this on PRs from forks https://github.com/actions/starter-workflows/issues/785#issuecomment-770195817

Reply:
Confirmed: this was able to run on the upstream main
```

replay localhosts Set { _c: Set { 'localhost', '127.0.0.1', '::1' } }
destination ***localhost:5984
source ***


  install
before replay localhosts Set {}
    theuserscouch
res [ '_replicator', '_users' ]
      1) should replicate theuserscouch
    new_corpus
res.body {
  ok: true,
  session_id: 'b47536ff7bb927d25cf2df97ced639c7',
  source_last_seq: 1,
  replication_id_version: 3,
  history: [
    {
      session_id: 'b47536ff7bb927d25cf2df97ced639c7',
      start_time: 'Tue, 02 Nov 2021 16:00:46 GMT',
      end_time: 'Tue, 02 Nov 2021 16:00:47 GMT',
      start_last_seq: 0,
      end_last_seq: 1,
      recorded_seq: 1,
      missing_checked: 1,
      missing_found: 1,
      docs_read: 1,
      docs_written: 1,
      doc_write_failures: 0
    }
  ]
}
res.body [ '_replicator', '_users', 'new_corpus', 'theuserscouch' ]
      2) should replicate new_corpus
    new_testing_corpus
      3) should replicate new_testing_corpus
    new_corpus_activity_feed
      - should replicate new_corpus_activity_feed
    new_user_activity_feed
      - should replicate new_user_activity_feed
    new_lexicon
res.body {
  ok: true,
  session_id: '772c41f88efcbe223ba3aadbd6c9beb3',
  source_last_seq: 10,
  replication_id_version: 3,
  history: [
    {
      session_id: '772c41f88efcbe223ba3aadbd6c9beb3',
      start_time: 'Tue, 02 Nov 2021 16:00:47 GMT',
      end_time: 'Tue, 02 Nov 2021 16:00:49 GMT',
      start_last_seq: 0,
      end_last_seq: 10,
      recorded_seq: 10,
      missing_checked: 11,
      missing_found: 11,
      docs_read: 11,
      docs_written: 11,
      doc_write_failures: 0
    }
  ]
}
res.body [
  '_replicator',
  '_users',
  'new_corpus',
  'new_testing_corpus',
  'theuserscouch'
]
res.body {
  ok: true,
  session_id: '8a073cbf697fe32582b1e68f7fc3e45b',
  source_last_seq: 16,
  replication_id_version: 3,
  source_last_seq: 1,
  replication_id_version: 3,
  history: [
    {
      session_id: 'cbe642fa13dceabb54cc66d1f2ec75e1',
      start_time: 'Tue, 02 Nov 2021 16:00:51 GMT',
      end_time: 'Tue, 02 Nov 2021 16:00:52 GMT',
      start_last_seq: 0,
      end_last_seq: 1,
      recorded_seq: 1,
      missing_checked: 2,
      missing_found: 2,
      docs_read: 2,
      docs_written: 2,
      doc_write_failures: 0
    }
  ]
}
res.body [
  '_replicator',
  '_users',
  'new_corpus',
  'new_lexicon',
  'new_testing_corpus',
  'theuserscouch'
]
      ✓ should replicate new_lexicon (137ms)
res.body [
  '_replicator',
  '_users',
  'new_corpus',
  'new_lexicon',
  'new_testing_corpus',
  'theuserscouch'
]
after replay localhosts Set { _c: Set { 'localhost', '127.0.0.1', '::1' } }


  1 passing (8s)
  2 pending
  4 failing
```